comm_norms.adoc

```diff
@@ -234,104 +282,138 @@ Remember to be gracious and thankful in your discussions with other project part
 === When issues become the subject of heated debate
 
 At times, the details of addressing a particular issue can cause tension or argument within the community.
+PD--"Argument" should either be made plural, or "an" should be inserted before it.
+
 While healthy and respectful debate is part of any thriving project—software or otherwise—tempers can flare easily, and (as has been well documented) people tend to behave with less civility online than they would in person.
+PD--What's your source for "as has been well documented"?
+
 If an issue has become especially contentious and discourse has become rude or inflammatory, restrict access to that issue for a stated period of time (say 24 to 48 hours) to allow people time to calm down, reflect, and state their argument in a more even-tempered and constructive manner.
 
 === Quick tips for filing issues well
+PD--The bullets below are pushing the limit on length.
 
-. Thank the people creating the software for their time and energy, especially if you are brand new to the project.
+. Thank the people creating the software for their time and energy, especially if you are new to the project.
 The individuals spending their (free) time creating free and open source software for you to use are also people who want to know their time is valued and their work appreciated.
+PD--Maybe "spare" instead of "free"?
+
 . Include as much information as you possibly can about the error you have encountered.
 If the project uses issue templates, fill it out as completely as possible.
-If you do not have the information requested or cannot determine how to get it yourself, then simply note what you have attempted to do in order to get the information.
+PD--There's an issue with number here. You've got templates (plural) and it (singular).
+
+If you do not have the information requested or cannot determine how to get it yourself, simply note what you have attempted to do in order to get the information.
 These details help mainters determine what they might need to do to assist you.
+PD--Did you mean "maintainers" here?
+
 . If a project does not use issue templates, look at other issues that have been "closed‒fixed" or at merged pull requests to see how other people have filed bug reports.
+PD--Make sure the number for "templates" here agrees with whatever you decide in the bullet above.
+
 If the issue was fixed, chances are quite good that you'll be able to use these historical artifacts as examples of the sort of information necessary for reproducing an error.
 Replicate what you find in these reports and add more detail as you are able.
 
 === Quick tips for responding to issues well
+PD--Like some of the other bulleted lists, the bullets here are rather long. Can you pare down the content any more or try another presentation?
```

Comment:
https://www.bbc.com/future/article/20180403-why-do-people-become-trolls-online-and-in-social-media